File: solutions/javascript/raindrops/1/raindrops.js
//
// This is only a SKELETON file for the 'Raindrops' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

export const convert = (raindropsNb) => {
  const divBySeven = raindropsNb%7===0
  const divByThree = raindropsNb%3===0
  const divByFive = raindropsNb%5===0

  let sound = ''
  sound = divByThree ? 'Pling' : sound
  sound = divByFive ? sound+'Plang' : sound
  sound = divBySeven ? sound+'Plong' : sound

  return (!divBySeven && !divByThree && !divByFive) ? raindropsNb.toString() : sound
};

File: solutions/go/raindrops/1/raindrops.go
package raindrops
import "fmt"

func Convert(number int) string {
	isDivibleByThree := number%3==0
    isDivibleByFive :=  number%5==0
    isDivibleBySeven := number%7==0
	res:=""
    if isDivibleByThree {
        res+="Pling"
    }

    if isDivibleByFive {
        res+="Plang"       
    }

    if isDivibleBySeven {
        res+="Plong"        
    }
	if res=="" {
        res+=fmt.Sprintf("%d",number)
    }
	return res
}
